themes/src/main/themes/VAADIN/themes/valo/components/_treegrid.scss

```diff
@@ -1,49 +1,43 @@
 @import "grid";
 
 /** Expander button visual - expanded */
-$tg-expander-char-expanded: '\f0d7' !default;
+$v-treegrid-expander-char-expanded: '\f0d7' !default;
 
 /** Expander button visual - collapsed */
-$tg-expander-char-collapsed: '\f0da' !default;
-
-/** Expander button width */
-$tg-expander-width: 10px !default;
-
-/** Expander button right side padding */
-$tg-expander-padding: 10px !default;
+$v-treegrid-expander-char-collapsed: '\f0da' !default;
 
 @mixin valo-treegrid ($primary-stylename: v-treegrid) {
 
   @include valo-grid($primary-stylename);
 
   // Expander with and item indentation constants
-  $indent: $tg-expander-width + $tg-expander-padding;
+  $indent:2ex !default;
```

Comment:
This variable is scoped by the mixin, so I’m not sure the `!default` keyword has any meaning here. It should be moved to the top level and be prefixed with `v-treegrid-`.

Also, why `ex`? That unit is just pretty much never used. `em` would be a safer choice as it is relative to the font size, whereas `ex` is relative to the font family. `1em` is a suitable default value.

Nit: there should be a space between the colon and the value.